# MPC parameters
predict_vars:
  x: 0
  z: 0
  dx: 100
  dz: 100
  att: 1
  datt: 1
  ddx: 0
  thrust_scaling: 0
  disturbance_torque: 0

update_vars:
  gyro: 0
  accel: 0
  baro: 0